### PR TITLE
Object#10378,  Edge Application Server Information

### PR DESCRIPTION
**Pull request by ezhaboy**
This LwM2M Object used for UE to obtain information about Edge Application Server that it shall communicate through an Edge Data Network, refer to [3GPP TS 23.558].

### CODE REVIEW

**Review by mkgillmore — CHANGES_REQUESTED** [review]
License is specified twice,  Please remove the redundant license declaration.

File: 10378.xml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+		<ObjectURN>urn:oma:lwm2m:x:10378</ObjectURN>
+		<LWM2MVersion>1.0</LWM2MVersion>
+		<ObjectVersion>1.0</ObjectVersion>
+		<MultipleInstances>Multiple</MultipleInstances>
+		<Mandatory>Optional</Mandatory>
+		<Resources>
+			<Item ID="0">
+				<Name>EAS ID</Name>
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
```
Comment (hannestschofenig) [inline 10378.xml:47]: Matt suggested to think about using multiple instances of this resource here.

**Comment by asoloway64** [comment]
I think we need to have a detailed discussion on the purpose/usage of this object. 3GPP TS 23.558 specifies a complete ecosystem for enabling an Application Client (AC) in the UE to access an Edge Application Server (EAS). This object does not take into account that entire ecosystem, but uses it as a justification to provide EAS contact information. I would counter that we can propose a cloud or edge application server information object, but it should not claim to be aligned with TS 23.558.

**Comment by ezhaboy** [comment]
> I think we need to have a detailed discussion on the purpose/usage of this object. 3GPP TS 23.558 specifies a complete ecosystem for enabling an Application Client (AC) in the UE to access an Edge Application Server (EAS). This object does not take into account that entire ecosystem, but uses it as a justification to provide EAS contact information. I would counter that we can propose a cloud or edge application server information object, but it should not claim to be aligned with TS 23.558.

Thank you for your comprehensive comments,  the purpose of this new Object is to provide EAS information to the LwM2M Client (as a lightweight EEC), leveraging LwM2M existing features.
In this use case, A LwM2M Server will interact with ECS and EES(s) to obtain EAS information through reference points EDGE-1 and EDGE-4 on behalf of the EEC (acting as a gateway), assisting EAS discovery and service continuity.
The Object refers terminology of [3GPP TS 23.558], it may not used to bring entire ecosystem of [3GPP TS 23.558], therefore the Object registered as company produced at current stage.

**Comment by asoloway64** [comment]
I now see that this is an Ericsson specific object. I have no objections.

File: 10378.xml (hunk 1 above)
Comment (ezhaboy) [reply 10378.xml:47]: @mkgillmore, @hannestschofenig as confirmed that three Resource Instance can be multiple.

**Comment by ezhaboy** [comment]
@hannestschofenig, @mkgillmore  I have to on hold registration due company internal process, sorry for that.

**Pull request closed without merging**